test/ccc/core/base/panel/legend/legend-panel-spec.js

```diff
@@ -0,0 +1,111 @@
+define([
+  'ccc/pvc',
+  'ccc/def',
+  'test/utils',
+  'test/data-1'
+], function(pvc, def, utils, datas) {
+
+  describe("legend-panel -", function() {
```

Comment:
Please remove all end of line punctuation from describe/it. The new karma reporter does not need this.